benchmarks/tpcds/prepare.py

```diff
@@ -53,6 +54,11 @@ def _stream_to_db(self, table):
                 f"psql {self.args.dsn} -c \"COPY {table} FROM STDIN "
                 f"WITH (FORMAT CSV, DELIMITER '|')\"")
 
+    def get_copy_cmds(self, table):
```

Comment:
This should be private method, should be named `_get_copy_cmds`

Reply:
i copied it from the htap benchmark. so if that should be the case then lets do that sort of cleanup in a separate PR?

Reply:
OK